### ISSUE TITLE
[Screen Reader-Help-Getting Started]Screen Reader is not providing the information about the ‘Starting tutorial’ loading message displayed on screen

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2545]
Pre-Requisite:Turn on Narrator(Win + Ctrl + Enter)

**User Experience:**
Users who depends on Screen reader will not be able to get information if Screen Reader will not provide the information about any loading message displayed on screen. 

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.

**Repro Steps:-**
1. Navigate to https://makecode.microbit.org/acc
2. Navigate to the Microbit section element and select Code control given on it.
3. Navigate to the Help control lying in the header section on the page and select it.
4. Navigate to 'Getting started' control and select it.
5. Verify that Screen Reader is providing the information about the ‘Starting tutorial’ loading message displayed on screen or not.

**Actual Result:-**
Screen Reader is not providing the information about the ‘Starting tutorial’ loading message displayed on screen.

**Expected Result:-**
Screen Reader should provide the information about the ‘Starting tutorial’ loading message displayed on screen.

**Note:** 
1.This issue exists for NVDA + FF.
2.This issue exists through out the application.

**MAS Reference -**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Suggested Fix:**
All the elements should be programmatically aligned correctly  
-      Paragraph elements should be tagged with <p> tag 
-      Link elements should have <a href> 
-      Buttons should have <a> or  <a role=”button”> 
-      Only interactive elements should be aligned with <a> tag, static elements on the page should not be made interactive by using <a> tag  
Reference: 
https://www.w3.org/WAI/WCAG20/quickref/?showtechniques=111%2C211%2C131#content-structure-separation-programmatic 

**Please refer the Attachement**
[MAS1.3.1_Help_Getting Started.zip](https://github.com/Microsoft/pxt/files/1179635/MAS1.3.1_Help_Getting.Started.zip)

**Comment by v-mesala** [comment]
Bug is verified on URL : https://makecode.microbit.org/acc bug is fixed .hence closed the bug and added the tag closed.
